### DISCUSSION TITLE
I just updated my VLAD diffusion; is the high-res fix gone and replaced with Second Pass?

### DISCUSSION THREAD

**Discussion by weirdlighthouse** [vladmandic/sdnext#1818] [Q&A] [answered]
****why is there need for Secondary sampler method? Is it for faster work flow? first use Euler a, then high-res fix it with a better sampler.****

![hfhfhfhfh](https://github.com/vladmandic/automatic/assets/140368252/f579ec60-17d9-4fa3-9a56-46499a43d7a3)

I
I
I

![fewwwwfwff](https://github.com/vladmandic/automatic/assets/140368252/444a292c-bcb2-4b92-b112-8ec19a5b1dfe)

****question also what does this do? any video or tip for this?****

**Comment by vladmandic** [comment, marked as answer]
> why is there need for Secondary sampler method? Is it for faster work flow? first use Euler a, then high-res fix it with a better sampler.

not necessarily faster, just different. 

> question also what does this do? any video or tip for this?

for original backend, only secondary cfg scale is implemented and it does exactly what cfg scale does for primary part of the workflow.
other settings are right now diffusers only, but will be eventually backported to original backend.
